src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs

```diff
@@ -51,6 +51,12 @@ private sealed class State
             public readonly TaskCompletionSource<uint> ConnectTcs = new TaskCompletionSource<uint>(TaskCreationOptions.RunContinuationsAsynchronously);
             public readonly TaskCompletionSource<uint> ShutdownTcs = new TaskCompletionSource<uint>(TaskCreationOptions.RunContinuationsAsynchronously);
 
+            // Note that there's no such thing as ressetable TCS, so we need reallocate the TCS every time we set the result.
```

Comment:
typo: `ressetable` => `resetable`

Reply:
Also nit: we don't need to reallocate "every time we set the result", right? We just can't reuse them.

Reply:
Fixed, reworded.